src/dependencies.js

```diff
@@ -21,13 +22,21 @@ const dependencyMap = {
   xtst: '(libXtst or libXtst6)'
 }
 
+async function getRpmVersion (logger, spawnProgram = spawn) {
```

Comment:
I'd rather add [`proxyquire`](https://www.npmjs.com/package/proxyquire) to `devDependencies` than add arguments that only get changed for testing purposes.
```suggestion
async function getRpmVersion (logger) {
```